test/SharedFx.UnitTests/SharedFxTests.cs

```diff
@@ -37,12 +37,13 @@ public async Task BaselineTest(SharedFxConfig config)
 
             var zipPath = Path.Combine(AppContext.BaseDirectory, zipName);
 
-            if (!Directory.Exists(AppContext.BaseDirectory + "unzipped"))
+            var tempPath = Path.GetTempPath();
+            if (!Directory.Exists(Path.Combine(tempPath, "unzipped")))
```

Comment:
This test will mess up if the `$env:Temp/unzipped` folder exists and contains something other than what `ZipFile` contains. Probably a good idea to use `Path.GetRandomFileName()` instead of the constant name.

Also, delete the directory if it already exists.

Reply:
Oh, I gave him bad advice to use `GetTempPath` because I was under the impression that the result was random rather than fixed.

Reply:
Okay, I'll incorporate this 👍